.gitignore

```diff
@@ -379,4 +379,6 @@ $RECYCLE.BIN/
 webserver/script/costbenefitanalysis/cases/test_files
 webserver/script/costbenefitanalysis/preprocessing_loadprofiles/inputs/weekdays/*
 webserver/script/costbenefitanalysis/preprocessing_loadprofiles/inputs/weekends/*
-webserver/script/costbenefitanalysis/preprocessing_loadprofiles/outputs/*
\ No newline at end of file
+webserver/script/costbenefitanalysis/preprocessing_loadprofiles/outputs/*
+webserver/app/redis-6.0.8
+webserver/app/redis-6.0.8.tar.gz
```

Comment:
this git ignore entry is a bit weird, doesn't hurt anything, but having the binary shouldn't be in the project dir in the first place.

Reply:
Thanks for the insight! I’ll re-evaluate how I’m installing Redis and revise this.